main.py

```diff
@@ -1,10 +1,55 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+
+    with open("players.json", "r") as file:
+        data = json.load(file)
+
+    for players_name, other_info in data.items():
```

Comment:
I think it's better to use simple singular `player`, because now it looks like plural, even id you meant `player's` here. You may also just get rid of `players_` part in all of the variables. It doesn't make much difference here.